modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita.adoc

```diff
@@ -199,7 +193,6 @@ Note: If you are using MySQL, add `?allowMultiQueries=true` to the URL. For exam
     Also, if you are migrating to Bonita 7.9+, you must upgrade your database server to MySQL 8.0, see <<mysql8,Migrating to Bonita 7.9+ using MySQL>> specific procedure below.
 ====
 
-. If you use a custom Look & Feel, xref:managing-look-feel.adoc[export] it, and then xref:managing-look-feel.adoc[restore the default Look & Feel].
 . Generally speaking, pausing (or not) the BPM services before migrating should not impact migration. There are however a few special cases:
```

Comment:
before migrating should not impact migration.  => before updating should not have any impact.

Reply:
Again, maybe treat it in another ticket